docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -19,79 +19,118 @@ This guide will walk you through the steps of doing this migration.
 
 ---
 
-## Prerequisites
+## Step 1: Take inventory of your existing Airflow usage
 
-To get started, you will need to install the `dagster` and `dagster-airflow` Python packages:
+Making migrations is complicated, there are lots of moving pieces and it will be important for you to go in knowing exactly how much scope and complexity you'll be tackling.
```

Comment:
Maybe something like:

"Before embarking on a migration it is important to understand the scope and complexity of your existing Airflow installation."